rpc/src/main/java/org/dst/rpc/core/common/constants/CodecConstants.java

```diff
@@ -0,0 +1,67 @@
+package org.dst.rpc.core.common.constants;
+
+import org.dst.rpc.core.exception.CodecException;
+
+
+public interface CodecConstants {
+
+  short MAGIC_HEAD = 9527;
+
+  int HEADER_LENGTH = 16/* bytes, not bits */;
```

Comment:
Could you introduce the protocol?
Given the information that we had discussed offline, there's no `HEADER_LENGTH` in my mind.

Reply:
HEADER_LENGTH is just a constants that will be used in rpc framework to know how large the protocol header is. It is not in protocol header.

Reply:
The full protocol header is in the DstCodec.class.